src/screens/SUE/SueReportScreen.tsx

```diff
@@ -225,6 +285,39 @@ export const SueReportScreen = ({ navigation }: { navigation: any }) => {
     }
   };
 
+  useLayoutEffect(() => {
+    if (storedValues) {
+      navigation.setOptions({
+        headerRight: () => (
+          <HeaderRight
+            {...{
+              onPress: () =>
+                Alert.alert(
+                  texts.sue.report.alerts.dataDeleteAlert.title,
+                  texts.sue.report.alerts.dataDeleteAlert.message,
+                  [
+                    { text: texts.sue.report.alerts.dataDeleteAlert.cancel },
+                    {
+                      text: texts.sue.report.alerts.dataDeleteAlert.ok,
+                      onPress: resetStoredValues,
+                      style: 'destructive'
+                    }
+                  ]
+                ),
+              navigation,
+              route,
+              withDelete: true
+            }}
+          />
+        )
+      });
+    } else {
+      navigation.setOptions({
+        headerRight: () => null
+      });
```

Comment:
is this else really necessary? what if there are no stored values and nothing wpuld be done?